### PR TITLE
Add SPV_QCOM_image_processing2

### PR DESCRIPTION
**Pull request by wooyoungqcom**
This is a follow-on extension to SPV_QCOM_image_processing.

This extension introduces a new set of operations for image processing, along with
a new capability and a new decoration.

operations:
OpImageBlockMatchWindowSADQCOM
OpImageBlockMatchWindowSSDQCOM
OpImageBlockMatchGatherSADQCOM
OpImageBlockMatchGatherSSDQCOM

capability:
TextureBlockMatch2QCOM

decoration
BlockMatchSamplerQCOM

### CODE REVIEW

File: extensions/QCOM/SPV_QCOM_image_processing2.asciidoc [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+Notice
+------
+
+Copyright (c) 2023 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- draft
```
Comment (bashbaug) [inline extensions/QCOM/SPV_QCOM_image_processing2.asciidoc:32]: Minor: Consider updating the status, unless this really is a spec draft.  I think we usually say "complete" or "shipping" or similar for most of the published and final extensions.

**Review by bashbaug — COMMENTED** [review]
Can you also please confirm that this extension should be revision "0"?

File: extensions/QCOM/SPV_QCOM_image_processing2.asciidoc (hunk 1 above)
Comment (wooyoungqcom) [reply extensions/QCOM/SPV_QCOM_image_processing2.asciidoc]: Thank you, Ben, for the comments.  I fixed the revision as well as the Status.

**Review by bashbaug — APPROVED** [review]
Merging as discussed in the January 10th teleconference.

**Pull request merged**